test-reports/TEST-tests.hfmagpsu.HfmagpsuTests-20170822144228.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
```

Comment:
These test files should not have been committed to the repository. Please remove them all.